_api-reference/ingest-apis/ingest-processors.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Ingest processors
+parent: Ingest APIs
+nav_order: 10
+has_children: true
+---
+
+# Ingest processors
+
+Ingest processors are a core component of [ingest pipelines]({{site.url}}{{site.baseurl}}/api-reference/ingest-apis/pipelines/), as they preprocess documents before indexing. For example, you can remove fields, extract values from text, convert data format, or append additional information.
+
+OpenSearch provides a standard set of ingest processors within your OpenSearch installation. For a list of processors available in OpenSearch, use the [nodes info]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/nodes-info/) API operation:
+
```

Comment:
@heemin32 What permissions are needed? Should we add those permissions to https://opensearch.org/docs/latest/security/access-control/api/?